airbyte-integrations/connectors/source-google-sheets/integration_tests/expected_records.txt

```diff
@@ -31,4003 +31,2003 @@
-{"stream":"Sheet6-4000-rows","data":{"ID":"2654","Name":"VLTTHBrSA"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2655","Name":"zoTgMCBpG"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2656","Name":"dJkUxyHcJ"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2657","Name":"WsDCfjVjR"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2658","Name":"IRqDptXBg"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2659","Name":"QmmhfvOaQ"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2660","Name":"wmDKwUAXv"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2661","Name":"tAELUDntc"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2662","Name":"FcoOLKkOD"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2663","Name":"YXSGUdvbv"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2664","Name":"mIXcxavSh"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2665","Name":"lTuBCtVib"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2666","Name":"ngFfZugdi"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2667","Name":"GHThybivy"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2668","Name":"RfTkuFnlB"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2669","Name":"SxDNtaCZo"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2670","Name":"YgdVWUVQD"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2671","Name":"nxGLSOWhs"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2672","Name":"HKmLHPARs"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2673","Name":"vqZLDvtil"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2674","Name":"yrjMPtsbJ"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2675","Name":"sBlpYnbND"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2676","Name":"vCwNEuhCf"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2677","Name":"JbnjNZama"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2678","Name":"gIJydagnM"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2679","Name":"XaMZnggHu"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2680","Name":"YcPoGucCx"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2681","Name":"DgLUpVADq"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2682","Name":"RNmvhcRUC"},"emitted_at":1673989572000}
-{"stream":"Sheet6-4000-rows","data":{"ID":"2683","Name":"KsGNtMSGo"},"emitted_at":1673989572000}
-{"stream"
```

Comment:
Why have we renamed the stream/reduced the number of rows?

Reply:
These changes were implemented when I was attempting to accelerate CAT tests during development. After the default step has been changed to `1_000_000`, the process will take the same amount of time, so I can rollback this change if needed.

Reply:
Not needed, I just wanted to understand. Thanks for the explanation. I'll resolve this